recipe/meta.yaml

```diff
@@ -1,46 +1,64 @@
-{% set version = "0.1.40" %}
-{% set sha256 = "385dfb965eca75164d256486c7cf9bae772d24144249fd18b9d15d3cffb70eea" %}
+{% set name = "python-debian" %}
+{% set version = "1.0.1" %}
 
 package:
-  name: python-debian
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/python-debian/python-debian-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 3ada9b83a3d671b58081782c0969cffa0102f6ce433fbbc7cf21275b8b5cc771
+  - url: https://salsa.debian.org/python-debian-team/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.bz2
+    sha256: c8f15572589a52e9486ec7fbcf9a2395577e97a8444ac2083cb9c7bf8c7ffc1a
+    folder: gl
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
     - python
     - wheel
-    - setuptools
+    - setuptools >=61
+    - setuptools_scm >=6.2
+    - tomli
```

Comment:
```suggestion
```
`setuptools_scm[toml]` expands to an empty value https://github.com/pypa/setuptools-scm/blob/main/pyproject.toml#L53. We can ignore it